iconservice/iconscore/icon_score_api_generator.py

```diff
@@ -109,7 +114,13 @@ def __generate_fallback_function(func_name: str, is_payable: bool, sig_info: 'Si
         info = dict()
         info[ScoreApiGenerator.__API_TYPE] = ScoreApiGenerator.__API_TYPE_FALLBACK
         info[ScoreApiGenerator.__API_NAME] = func_name
-        info[ScoreApiGenerator.__API_INPUTS] = ScoreApiGenerator.__generate_inputs(dict(sig_info.parameters))
+
+        if len(sig_info.parameters) > 1:
+            raise InvalidParamsException("can't have params")
+
+        if sig_info.return_annotation is not None:
+            if sig_info.return_annotation is not Signature.empty:
+                raise InvalidParamsException("can't have return value")
 
         if is_payable:
```

Comment:
This check is unnecessary if my suggesting is accepted.

Reply:
I think payable field is necessary.
if I want to override payable without payable decorator, it would also be hide in SCORE api.

Reply:
I misunderstood about it.
only remove this code is good.